Tasks/PublishBuildDropV0/task.json

```diff
@@ -0,0 +1,42 @@
+{
+  "id": "ECDC45F6-832D-4AD9-B52B-EE49E94659BE",
+  "name": "PublishBuildDrop",
+  "friendlyName": "Publish Build Drop",
+  "description": "Publish Build Drop",
+  "helpMarkDown": "",
+  "category": "Utility",
+  "author": "Microsoft Corporation",
+  "version": {
+    "Major": 0,
+    "Minor": 137,
+    "Patch": 0
+  },
+  "groups": [],
+  "demands": [],
+  "inputs": [
+    {
+      "name": "artifactname",
+      "type": "string",
+      "label": "The name of this artifact",
+      "defaultValue": "",
```

Comment:
i thought we have a default artifact name "drop" today, do we?

Reply:
added